cgwire/kitsu
# Fixes and improvements

**Problem**

QA:

* SASS utils are not used by every component
* Color variables are not used enough
* Folders are a bit messy

Bugs:

* Task list column sizes don't adapt to data.
* Video loading spinner is not properly displayed
* Task search queries are not saved
* Deleting the last playlist keeps the playlist modal open

Enhancements:

* .m4v files are not supported as previews
* It is not possible to drop files directly on the comment area 


**Solution**

QA:

* Configure each component to use SASS as CSS framework
* Replace most of color values by color variables
* Put page components in a dedicated folder

Bugs:

* Resize task list headers, each time the list changes
* Hide videos when loading spinner is displayed
* Save mandatory entity type field when saving a filter for the task list.
* Handle properly modal displaying when there are no more playlists

Enhancements:

* Add .m4v files to accepted extensions for preview upload
* Listen to drop events to allow to drop files directly on the comment area
